src/Widgets/ViewEditStack.vala

```diff
@@ -76,12 +76,19 @@ public class ENotes.ViewEditStack : Gtk.Overlay {
                 return;
             }
         }
-
+        
         editor.save_file ();
         current_page = PageTable.get_instance ().get_page (page.id);
         editor.current_page = current_page;
         viewer.load_page (current_page);
 
+        
+        
+        Notebook ntbook = ENotes.NotebookTable.get_instance().load_notebook_data(current_page.notebook_id);
+        string notebook_title = ntbook.name;
+        ENotes.Headerbar.get_instance ().set_title (page.name, notebook_title);
```

Comment:
1. Too many empty lines around here.
2. Use `var notebook` (if there aren't any conflicts) instead of `Notebook ntbook`
3. No need to save the title to it's own variable. You can just pass notebook.name to the `set_title function`